Makefile

```diff
@@ -36,6 +36,11 @@ index.js.tmp: index.js
 	sed -e '/^[/][/]:/ s,\($(SED_WORD_START)[[:alnum:]]*\),<a href="#\1">\1</a>,g' -e 's,^//:,//.,' '$<' >'$@'
 
 
+.PHONY: bench
+bench:
+	node bench
```

Comment:
This assumes that `npm install` has been run in the __bench__ directory. Shall we add something like `cd bench && $(NPM) install` to the `setup` target?